app/models/service_template_orchestration.rb

```diff
@@ -3,6 +3,19 @@ class ServiceTemplateOrchestration < ServiceTemplate
 
   before_save :remove_invalid_resource
 
+  def self.create_catalog_item(options, _auth_user = nil)
+    transaction do
+      create(options.except(:config_info)).tap do |service_template|
+        config_info = validate_config_info(options)
+
+        service_template.orchestration_template = OrchestrationTemplate.find(config_info[:template_id])
```

Comment:
We want to accept `:template_id` or `:template`. The later points to an object that can be directly assigned. Same to manager. Need to update the validate method too.